hs_geo_raster_resource/receivers.py

```diff
@@ -9,7 +9,7 @@
 # to populate create-resource.html template page
 @receiver(pre_create_resource, sender=RasterResource)
 def raster_pre_create_resource_trigger(sender, **kwargs):
-    if(sender is RasterResource):
+    if sender is RasterResource:
```

Comment:
If statement is not needed as the decorator already has the restriction (sender=RasterResource)